### PR TITLE
add package json and express

### PR DESCRIPTION
**Pull request by zakvyn**

### CODE REVIEW

**Comment by fzhougithub** [comment]
Should I just reply here?

**Comment by zakvyn** [comment]
I will show u

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "stock",
+  "version": "0.0.0",
+  "description": "Service analize stock trends",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 0"
+  },
+  "repository": {
```
Comment (fzhougithub) [inline package.json]: test

**Comment by fzhougithub** [comment]
:+1:

**Pull request closed without merging**